lib/pact/matching_rules/jsonpath.rb

```diff
@@ -49,10 +48,10 @@ def initialize(path)
             @path.last << token
           end
         end
+      end
 
-        def to_s
-          path.join
-        end
+      def to_s
```

Comment:
This appeared to be mistakenly nested inside the initializer, but I could be misunderstanding what was intended...

Reply:
I have no idea either, it was copy/pasted from another library.